### PR TITLE
Feature/wfe 2075

### PR DESCRIPTION
**Pull request by dbreuer**
Change:
- Workflow steps section style (color, background) update.
- Badge style update.

### CODE REVIEW

File: source/stylesheets/_tab-workflows.scss.erb [hunk 1]
```diff
@@ -1210,7 +1232,8 @@ header.sticky section.manage-workflows {
 										li.input.required-empty {
 											
 											.input-info textarea {
-												border-color: $def-danger;
+
```
Comment (doomsayer13) [inline source/stylesheets/_tab-workflows.scss.erb]: please remove the empty line

File: source/stylesheets/_tab-workflows.scss.erb [hunk 2]
```diff
@@ -1019,14 +1029,15 @@ header.sticky section.manage-workflows {
 											@include padding-y(20px);
 											padding-left: $leftpadding;
 											padding-right: $rightpadding;
-											background-color: $def-ultralightgrey;
+											border-top: 1px solid #ccc;
+											color: $def-mediumdarkgrey;
+
```
Comment (doomsayer13) [inline source/stylesheets/_tab-workflows.scss.erb]: please remove the empty line

File: source/stylesheets/_tab-workflows.scss.erb [hunk 3]
```diff
@@ -965,7 +965,16 @@ header.sticky section.manage-workflows {
 									}
 
 									&:not(.main) {
-										background-color: $def-ultralightgrey;
+										$margin: 10px;
+										$borderwidth: 1px;
+										overflow: hidden;
+										margin: $margin;
+										border-radius: 5px;
+										border: $borderwidth solid #ccc;
```
Comment (erosdome) [inline source/stylesheets/_tab-workflows.scss.erb]: There is a constant for `#ccc`, it is `$def-mediumlightgrey`.

File: source/stylesheets/_tab-workflows.scss.erb [hunk 4]
```diff
@@ -1019,14 +1029,15 @@ header.sticky section.manage-workflows {
 											@include padding-y(20px);
 											padding-left: $leftpadding;
 											padding-right: $rightpadding;
-											background-color: $def-ultralightgrey;
+											border-top: 1px solid #ccc;
+											color: $def-mediumdarkgrey;
```
Comment (erosdome) [inline source/stylesheets/_tab-workflows.scss.erb]: There is a constant for `#ccc`, it is `$def-mediumlightgrey`.

**Pull request merged**